views/Testimonials.tsx

```diff
@@ -1,12 +1,19 @@
 import React from 'react'
 import FamewallEmbed from 'react-famewall'
+import { getPreferredTheme } from '../utils/theme'
 
 const Testimonials = () => {
+	const [theme, setTheme] = React.useState('light');
+
+	React.useEffect(() => {
+		setTheme(getPreferredTheme());
+	}, []);
```

Comment:
use function defined in utils/theme? Low LOGAF

Reply:
Here, I am not just setting the theme, I also want the word 'light' or 'dark' so that I can pass it on to the `FamewallEmbed` component as a prop.